content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -27,6 +27,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
 * Swap disabled. You **MUST** disable swap in order for the kubelet to work properly.
+    * `sudo swapoff -a`
```

Comment:
```suggestion
    * For example: `sudo swapoff -a` (and make sure to make the change permanent)
``` 
There is no one consistent way to disable swap space on nodes. For example, you can also define a `.swap` unit in systemd, or have an EFI partition of type `0657fd6d-a4ab-43c4-84e5-0933c84b4f4f` that autodiscovers. The suggestion here would not make permanent changes which is likely to surprise readers.

Reply:
updated in https://github.com/kubernetes/website/pull/40654/commits/88cb81f65ea34125c4f5b127d07cf12f622e7fdd